src/assets/sass/protocol/templates/_card-layout.scss

```diff
@@ -0,0 +1,204 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../includes/lib';
+
+/* -------------------------------------------------------------------------- */
+// Card grid layout classes.
+
+/* -------------------------------------------------------------------------- */
+// 5 card hero
+// On small screens cards stack vertically and span full container width.
+// On medium sized screens the 1st card spans full width, and the remaining 4 cards span half widths.
+// On large screens the 1st card spans 2 thirds, and the remaining cards span 1 third.
+.mzp-l-card-hero {
+    @media #{$media-sm} {
+        @include clearfix;
+
+        .mzp-c-card {
+            @include bidi((
+                (float, left, right),
+                (margin-left, 0, $margin-xl),
+                (margin-right, $margin-xl, 0),
+            ));
+            width: calc(50% - (#{$margin-xl} / 2));
+
+            &:nth-child(odd) {
+                margin-left: 0;
+                margin-right: 0;
+            }
+
+            &.mzp-c-card-large {
+                width: 100%;
+            }
+        }
+    }
+
+    @media #{$media-lg} {
+        .mzp-c-card {
+            width: calc(33.3% - (#{$margin-xl} - (#{$margin-xl} / 3)));
+
+            &:nth-child(odd) {
+                @include bidi((
+                    (margin-left, 0, $margin-xl),
+                    (margin-right, $margin-xl, 0),
+                ));
+            }
+
+            &:nth-child(2),
+            &:last-child {
+                margin-right: 0;
```

Comment:
I think you can use the bidi mixin here as well. `@include bidi(((margin-right, 0, margin-left, 0),));`

If I understand, the mixin behaves a little differently when there are four values instead of three: https://github.com/mozilla/protocol/blob/master/src/assets/sass/protocol/includes/mixins/_bidi.scss#L20

Reply:
Ah! i struggled with this and didn't grok the extra value. Thanks for the pointer 👍

Reply:
I'm not sure I fully grok it either... I swiped that mixin from MDN. It almost seems to be doing too much. I wonder if it could be simplified.